src/sentry/ratelimits/utils.py

```diff
@@ -27,6 +28,15 @@
     "members:org-invite-to-email": {"limit": 10, "window": 3600 * 24},
 }
 
+_CONCURRENT_RATE_LIMITER = ConcurrentRateLimiter()
+
+
+def concurrent_limiter() -> ConcurrentRateLimiter:
```

Comment:
Why is this needed?

Reply:
It's a best practice to put global state behind a function call.